src-python/lab-service/traffexam/kilda/traffexam/action.py

```diff
@@ -66,6 +67,33 @@ def _encode(self, entry):
         return Ether(src=entry.mac_address, dst=self.lldp_bcast_mac) / payload
 
 
+class LACPPush(Abstract):
+    def __call__(self, iface, push_entry):
+        pkt = self._encode(push_entry)
+
+        with self._network_namespace_context:
+            sendp(pkt, iface=iface.name, verbose=False, promisc=False)
+
+    def _encode(self, entry):
+        # entry, containing 8 boolean fields -> string representation of binary number, e.g. '10011010'
+        actor_state_binary = ""
+        for field in [entry.expired,
+                      entry.defaulted,
+                      entry.distributing,
+                      entry.collecting,
+                      entry.synchronization,
+                      entry.aggregation,
+                      entry.lacp_timeout,
+                      entry.lacp_activity]:
+            actor_state_binary += self._bool_to_str(field)
+        # Kilda ignores everything except lacp actor state. If this changes - expand this method with more fields
```

Comment:
This is not true. Kilda stores info about partner's ports, system_id, keys, etc 
[docker/db-migration/migrations/024-add-lacp-partner-class.yaml](https://github.com/telstra/open-kilda/pull/5040/files#diff-b4b905fcb50b0c5bced1c9f4b963f9e7cda403a9b9efe534f08e96ac4ad94c58)